src/dmd/dtemplate.d

```diff
@@ -623,14 +626,34 @@ extern (C++) final class TemplateDeclaration : ScopeDsymbol
                                 if (auto ti = ad.type.isTypeIdentifier())
                                 {
                                     if (ti.idents.length == 0 &&
-                                        ti.ident is ttp.ident)
+                                        ti.ident is ttp.ident &&
+                                        ti.mod == 0)
                                     {
+                                        //printf("found isAliasSeq %s %s\n", s.toChars(), ad.type.toChars());
                                         isAliasSeq = true;
                                     }
                                 }
                             }
                         }
                     }
+                    else if (auto ttp = (*parameters)[0].isTemplateTypeParameter())
```

Comment:
This is unused, merge it with the above branch as it is almost identical.

Reply:
They are different, and 639 is used.

Reply:
I said almost identical. Where is it used? I can't see it.

Reply:
Its still only used for an identity comparison.

Reply:
I don't really get what you mean.

As far as merging it with the previous code, I tend to not do that in initial PRs because it tends to tangle the logic. Better to do that in later refactoring pass.